src/vdaf/prio2.rs

```diff
@@ -22,7 +22,7 @@ use std::{
 };
 
 /// The Prio2 VDAF. It supports the same measurement type as
-/// [`Prio3Aes128CountVec`](crate::vdaf::prio3::Prio3Aes128CountVec) but uses the proof system
+/// [`Prio2Aes128CountVec`](crate::vdaf::prio3::Prio2Aes128CountVec) but uses the proof system
```

Comment:
Was this intentional? There's no such thing as `Prio2Aes128CountVec` AFAIK.

Reply:
Indeed, this is causing the docs build to fail.

Reply:
Whoopsie daisy!